src/main/java/org/reaktivity/nukleus/http/internal/stream/ClientConnectReplyStream.java

```diff
@@ -958,12 +807,21 @@ private void handleWindow(
 
         if (slotIndex != NO_SLOT)
         {
-            decodeBufferedData();
+            MutableDirectBuffer slot = factory.bufferPool.buffer(slotIndex);
+            decode(slot, 0, slotOffset);
+            if (slotOffset == 0 && endDeferred)
```

Comment:
This should really be `if (slotIndex == NO_SLOT && endDeferred)`, right?

Reply:
I guess that works too.